operatorconfig/driverconfig/powerflex/v2.7.0/controller.yaml

```diff
@@ -52,6 +52,9 @@ rules:
   - apiGroups: [""]
     resources: ["secrets"]
     verbs: ["get", "list"]
+  - apiGroups: ["volumegroup.storage.dell.com"]
```

Comment:
@sakshi-garg1 Is there anything to do with this sidecar for resiliency module?

Reply:
removed this code